projects/igniteui-angular/src/lib/time-picker/time-picker.component.ts

```diff
@@ -204,7 +206,10 @@ export class IgxTimePickerComponent implements
      * An accessor that returns the label of ok button.
     */
     get okButtonLabel(): string {
-        return this._okButtonLabel || this.resourceStrings.igx_time_picker_ok;
+        if (this._okButtonLabel === null) {
```

Comment:
Do you actually need to change this code?

Reply:
Yes, with the previous implementation there was no way to remove the buttons via setting the label to an empty string, as it would default to the string from the resource file.